.gitignore

```diff
@@ -1,2 +1,115 @@
 composer.lock
-vendor
\ No newline at end of file
+vendor
+
+.idea
```

Comment:
I guess, this content do not relate to lib. Please, add that to your global git ignore and clean up fron lib repo.